dictionaries/paths.dic

```diff
@@ -0,0 +1,10 @@
+coursedetail
+coursetable
+fileviewer
+iplus
+ischoolplus
```

Comment:
The ntut_terms.dic file already contains the term "iSchool". It seems that the terms "iplus" and "ischoolplus" have the same meaning, so I added them to the paths.dic file so that they will be removed in the future. From now on, we should only use "iSchool" to refer to the website at the link <https://istudy.ntut.edu.tw/>.